Makefile

```diff
@@ -524,8 +526,20 @@ out/linters/golangci-lint-$(GOLINT_VERSION):
 .PHONY: lint
 ifeq ($(MINIKUBE_BUILD_IN_DOCKER),y)
 lint:
-	docker run --rm -v $(pwd):/app -w /app golangci/golangci-lint:$(GOLINT_VERSION) \
-	golangci-lint run ${GOLINT_OPTIONS} --skip-dirs "cmd/drivers/kvm|cmd/drivers/hyperkit|pkg/drivers/kvm|pkg/drivers/hyperkit" ./...
+	mkdir -p ./out/linters
+	@if [ ! -f ./out/linters/yq ] ; then \
+		wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/${YQ_BINARY_NAME} -O ./out/linters/yq && chmod +x ./out/linters/yq; \
+	fi
+	
+	./out/linters/yq eval \
+	'.linters.exclusions.paths += [ "cmd/drivers/kvm", "cmd/drivers/hyperkit", "pkg/drivers/kvm", "pkg/drivers/hyperkit" ]' \
+	-i .golangci.yaml  &&  echo "yq add data ok"  
+	
+	docker run --rm -v `pwd`:/app -w /app golangci/golangci-lint:$(GOLINT_VERSION) \
+	bash -c "apt update && apt -y install libvirt-dev && golangci-lint run ${GOLINT_OPTIONS} ./..." 
```

Comment:
did we really need to do apt update and install libvirt ?

Reply:
Currently this code is still under debugging. I submit this PR mainly for the convenience of discussion. I will merge and submit it later. Then notify you again.